content/content-management/types.md

```diff
@@ -48,7 +48,7 @@ The following examples take you stepwise through creating a new type layout for
 +++
 title = My First Event
 date = "2016-06-24T19:20:04-07:00"
-description = "Today is my 36th birthday. My how time flies."
+description = "Today is my 36th birthday. How time flies."
```

Comment:
Damn, but I've never heard this phrase before. Thanks for the clarification Ryan :wink: